ambassador/ambassador/mapping.py

```diff
@@ -122,6 +122,53 @@ def get(self, key, *args):
         else:
             return self.attrs.get(key)
 
+    def save_cors_element(self, cors_key, route_key, route):
+        """If self.get('cors')[cors_key] exists, and
+        - is a list, e.g. ["1","2","3"], then route[route_key] is set as "1, 2, 3"
+        - is something else, then set route[route_key] as it is
+
+        :param cors_key: key to exist in self.get('cors'), i.e. Ambassador's config
+        :param route_key: key to save to in envoy's cors configuration
+        :param route: envoy's cors configuration
+        """
+        cors = self.get('cors')
+        if cors.get(cors_key) is not None:
```

Comment:
Won't this raise exceptions if `cors_key` isn't present in `cors`? (This applies to many places here... sorry I missed it last time!)

Reply:
@kflynn will it? Just tried -
```python
>>> a={"a":1,"b":2}
>>> if a.get("c") is not None:
...     print("hi")
... 
```
and it didn't raise an exception even though "c" is not present in dict a.

Reply:
...just rechecked the docs and I don't know WTF I was thinking. :) :) You're right.